src/Symfony/Component/Security/Http/AccessToken/AccessTokenHandlerInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\AccessToken;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+/**
+ * The token handler retrieves the user identifier using the token.
+ * Its responsibility is to
+ *  - load the token
+ *  - check the token (revocation, expiration time, digital signature...)
+ *  - return the user ID associated to it.
```

Comment:
```suggestion
 *  - return the user identifier associated to it.
```

so ppl do not take it as a database id column for example